### DISCUSSION TITLE
Galaxy brain

### DISCUSSION THREAD

**Discussion by Ysrbolles** [Schweinepriester/github-profile-achievements#193] [Q&A] [answered]
Make sure to star my repo https://github.com/Ysrbolles/Darkly-42Network and i will give your the answer

**Comment by mikekpl** [comment, marked as answer]
Great Repo! Sending stars. Hope you can update it @Ysrbolles

**Reply by Ysrbolles** [reply]
Ohh thank you. Yes sure.

**Comment by Thulana-Thidaswin** [comment]
All right done, looks good

**Comment by atiqisrak** [comment]
https://github.com/atiqisrak/my-react-three/discussions

I need it too.
